frontend/ui/src/components/datetime-picker.tsx

```
@@ -0,0 +1,623 @@
+// Make it a standalone component
+
+const TimePicker = () => {
+  const { value, onValueChange, Time, onTimeChange } = useSmartDateInput();
+  const [activeIndex, setActiveIndex] = React.useState(-1);
+  const timestamp = 15;
+
+  const formateSelectedTime = React.useCallback(
+    (time: string, hour: number, partStamp: number) => {
+      onTimeChange(time);
+
+      let newVal = value ? new Date(value) : new Date();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'newVal' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)